### PR TITLE
ABA-6: Round off lifting insert to 1”

### PR DESCRIPTION
**Pull request by sridhar-codeindian**
#7

### CODE REVIEW

**Review by satishCodeIndians — COMMENTED** [review]
Under review

File: AutoCADAutomation/AutoCADAutomation/CollectionHelper.cpp [hunk 1]
```diff
@@ -349,6 +352,42 @@ void CollectionHelper::CollectLiftDimensions(AcDbEntity* entity)
...
+			CollectLinesWithAnAngle(entity, vecHorDimLiftPlaceHolders, vecVerDimLiftPlaceHolders);
+		if (wcscmp(entity->isA()->name(), L"AcDbRotatedDimension"))
+		{
+			if (wcscmp(entity->isA()->name(), L"AcDbText") == 0)
+			{
+				AcDbText* text = (AcDbText*)entity;
+				if (text)
+				{
+					if (wcscmp(text->textString(), L"(LIFT INSERTS)"))
+						entity->erase();
```
Comment (satishCodeIndians) [inline AutoCADAutomation/AutoCADAutomation/CollectionHelper.cpp:369]: Why are we deleting the dimensions here ?

Comment (sridhar-codeindian) [reply AutoCADAutomation/AutoCADAutomation/CollectionHelper.cpp]: Removed extra\unnecessary code

File: AutoCADAutomation/AutoCADAutomation/CollectionHelper.cpp [hunk 2]
```diff
@@ -349,6 +352,42 @@ void CollectionHelper::CollectLiftDimensions(AcDbEntity* entity)
...
+				}
+			}
+			else
+				entity->erase(); // remove the dimensions after collecting them
```
[the remark this answers is not in the document]
Comment (sridhar-codeindian) [reply AutoCADAutomation/AutoCADAutomation/CollectionHelper.cpp]: Removed extra\unnecessary code and committed the changes

**Review by satishCodeIndians — APPROVED** [review]

**Pull request merged**